great-idea/index.html

```diff
@@ -14,10 +14,98 @@
   <![endif]-->
 </head>
 
-  <body>
+  <body>  
+    <div class = "container">
+      <header>
+    <nav>
+  <a href ="file:///C:/Users/e0149196/Desktop/Personal%20Info/Lambda%20fullstack/Git/UI-III-Flexbox-master/UI-III-Flexbox-master/great-idea/services.html">Services</a>
+  <a href ="file:///C:/Users/e0149196/Desktop/Personal%20Info/Lambda%20fullstack/Git/UI-III-Flexbox-master/UI-III-Flexbox-master/great-idea/product.html">Product</a>
+      <a href ="#">Vision</a>
+          <a href ="#">Features</a>
+              <a href ="#">About</a>
+                  <a href ="#">Contact</a>
+  </nav>
 
-    <!-- Copy and paste your HTML from the first UI project here -->
-      
-    
+  <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+</header>
+
+<section class="sub">
+  
+  <h1>Innovation<br>
+  On <br>
+  Demand</h1> <br>
+
+  <h3>Get Started</h3>
+
+  <img src="img/header-img.png" alt="Image of a code snippet.">
+
+  <div class = "line">
+
+</section> 
+
+<div  class = "one">
+  <h6>Features</h6>
+  <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+<br>
+
+
+  <h3>About</h3>
+
+  <div class = "two">
+  <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+<br>
+<br>
+<br>
+<br>
+  <img class="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
+  <br>
+  <br>
+
+  <div class = "three">
+
+    <h4>Services</h4>
+  <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+
+  
+  <h5>Product</h5>
+
+ <div class = "four">
+  <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+
+  <div class = "five">
+
+  <h3>Vision</h3>
+  <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+  <br>
+
+  <br>
+
+
+     
+  <div class = "contact-section">
+<h2>Contact</h2>
+<address>
+<br>
+  123 Way 456 Street <br>
+  Somewhere, USA
+  <br>
+  <br>
+  1 (888) 888-8888
+  <br>
+  <br>
+  sales@greatidea.io
+  <br>
+  <br>
+</address>
```

Comment:
Something you could do to add a bit more interactivity here with the contact information is wrap the telephone number and email in anchor elements. Check out the address example from [DevDocs](https://devdocs.io/html/element/address).